// @typescript-eslint/scope-manager/dist/lib/dom.js
"use strict";
// THIS CODE WAS AUTOMATICALLY GENERATED
// DO NOT EDIT THIS CODE BY HAND
// RUN THE FOLLOWING COMMAND FROM THE WORKSPACE ROOT TO REGENERATE:
// npx nx generate-lib @typescript-eslint/scope-manager
Object.defineProperty(exports, "__esModule", { value: true });
exports.dom = void 0;
const base_config_1 = require("./base-config");
exports.dom = {
    AddEventListenerOptions: base_config_1.TYPE,
    AesCbcParams: base_config_1.TYPE,
    AesCtrParams: base_config_1.TYPE,
    AesDerivedKeyParams: base_config_1.TYPE,
    AesGcmParams: base_config_1.TYPE,
    AesKeyAlgorithm: base_config_1.TYPE,
    AesKeyGenParams: base_config_1.TYPE,
    Algorithm: base_config_1.TYPE,
    AnalyserOptions: base_config_1.TYPE,
    AnimationEventInit: base_config_1.TYPE,
    AnimationPlaybackEventInit: base_config_1.TYPE,
    AssignedNodesOptions: base_config_1.TYPE,
    AudioBufferOptions: base_config_1.TYPE,
    AudioBufferSourceOptions: base_config_1.TYPE,
    AudioConfiguration: base_config_1.TYPE,
    AudioContextOptions: base_config_1.TYPE,
    AudioNodeOptions: base_config_1.TYPE,
    AudioProcessingEventInit: base_config_1.TYPE,
    AudioTimestamp: base_config_1.TYPE,
    AudioWorkletNodeOptions: base_config_1.TYPE,
    AuthenticationExtensionsClientInputs: base_config_1.TYPE,
    AuthenticationExtensionsClientOutputs: base_config_1.TYPE,
    AuthenticatorSelectionCriteria: base_config_1.TYPE,
    BiquadFilterOptions: base_config_1.TYPE,
    BlobEventInit: base_config_1.TYPE,
    BlobPropertyBag: base_config_1.TYPE,
    CSSStyleSheetInit: base_config_1.TYPE,
    CacheQueryOptions: base_config_1.TYPE,
    CanvasRenderingContext2DSettings: base_config_1.TYPE,
    ChannelMergerOptions: base_config_1.TYPE,
    ChannelSplitterOptions: base_config_1.TYPE,
    CheckVisibilityOptions: base_config_1.TYPE,
    ClientQueryOptions: base_config_1.TYPE,
    ClipboardEventInit: base_config_1.TYPE,
    ClipboardItemOptions: base_config_1.TYPE,
    CloseEventInit: base_config_1.TYPE,
    CompositionEventInit: base_config_1.TYPE,
    ComputedEffectTiming: base_config_1.TYPE,
    ComputedKeyframe: base_config_1.TYPE,
    ConstantSourceOptions: base_config_1.TYPE,
    ConstrainBooleanParameters: base_config_1.TYPE,
    ConstrainDOMStringParameters: base_config_1.TYPE,
    ConstrainDoubleRange: base_config_1.TYPE,
    ConstrainULongRange: base_config_1.TYPE,
    ConvolverOptions: base_config_1.TYPE,
    CredentialCreationOptions: base_config_1.TYPE,
    CredentialPropertiesOutput: base_config_1.TYPE,
    CredentialRequestOptions: base_config_1.TYPE,
    CryptoKeyPair: base_config_1.TYPE,
    CustomEventInit: base_config_1.TYPE,
    DOMMatrix2DInit: base_config_1.TYPE,
    DOMMatrixInit: base_config_1.TYPE,
    DOMPointInit: base_config_1.TYPE,
    DOMQuadInit: base_config_1.TYPE,
    DOMRectInit: base_config_1.TYPE,
    DelayOptions: base_config_1.TYPE,
    DeviceMotionEventAccelerationInit: base_config_1.TYPE,
    DeviceMotionEventInit: base_config_1.TYPE,
    DeviceMotionEventRotationRateInit: base_config_1.TYPE,
    DeviceOrientationEventInit: base_config_1.TYPE,
    DisplayMediaStreamOptions: base_config_1.TYPE,
    DocumentTimelineOptions: base_config_1.TYPE,
    DoubleRange: base_config_1.TYPE,
    DragEventInit: base_config_1.TYPE,
    DynamicsCompressorOptions: base_config_1.TYPE,
    EcKeyAlgorithm: base_config_1.TYPE,
    EcKeyGenParams: base_config_1.TYPE,
    EcKeyImportParams: base_config_1.TYPE,
    EcdhKeyDeriveParams: base_config_1.TYPE,
    EcdsaParams: base_config_1.TYPE,
    EffectTiming: base_config_1.TYPE,
    ElementCreationOptions: base_config_1.TYPE,
    ElementDefinitionOptions: base_config_1.TYPE,
    ErrorEventInit: base_config_1.TYPE,
    EventInit: base_config_1.TYPE,
    EventListenerOptions: base_config_1.TYPE,
    EventModifierInit: base_config_1.TYPE,
    EventSourceInit: base_config_1.TYPE,
    FilePropertyBag: base_config_1.TYPE,
    FileSystemFlags: base_config_1.TYPE,
    FileSystemGetDirectoryOptions: base_config_1.TYPE,
    FileSystemGetFileOptions: base_config_1.TYPE,
    FileSystemRemoveOptions: base_config_1.TYPE,
    FocusEventInit: base_config_1.TYPE,
    FocusOptions: base_config_1.TYPE,
    FontFaceDescriptors: base_config_1.TYPE,
    FontFaceSetLoadEventInit: base_config_1.TYPE,
    FormDataEventInit: base_config_1.TYPE,
    FullscreenOptions: base_config_1.TYPE,
    GainOptions: base_config_1.TYPE,
    GamepadEventInit: base_config_1.TYPE,
    GetAnimationsOptions: base_config_1.TYPE,
    GetNotificationOptions: base_config_1.TYPE,
    GetRootNodeOptions: base_config_1.TYPE,
    HashChangeEventInit: base_config_1.TYPE,
    HkdfParams: base_config_1.TYPE,
    HmacImportParams: base_config_1.TYPE,
    HmacKeyAlgorithm: base_config_1.TYPE,
    HmacKeyGenParams: base_config_1.TYPE,
    IDBDatabaseInfo: base_config_1.TYPE,
    IDBIndexParameters: base_config_1.TYPE,
    IDBObjectStoreParameters: base_config_1.TYPE,
    IDBTransactionOptions: base_config_1.TYPE,
    IDBVersionChangeEventInit: base_config_1.TYPE,
    IIRFilterOptions: base_config_1.TYPE,
    IdleRequestOptions: base_config_1.TYPE,
    ImageBitmapOptions: base_config_1.TYPE,
    ImageBitmapRenderingContextSettings: base_config_1.TYPE,
    ImageDataSettings: base_config_1.TYPE,
    ImageEncodeOptions: base_config_1.TYPE,
    ImportMeta: base_config_1.TYPE,
    InputEventInit: base_config_1.TYPE,
    IntersectionObserverEntryInit: base_config_1.TYPE,
    IntersectionObserverInit: base_config_1.TYPE,
    JsonWebKey: base_config_1.TYPE,
    KeyAlgorithm: base_config_1.TYPE,
    KeyboardEventInit: base_config_1.TYPE,
    Keyframe: base_config_1.TYPE,
    KeyframeAnimationOptions: base_config_1.TYPE,
    KeyframeEffectOptions: base_config_1.TYPE,
    LockInfo: base_config_1.TYPE,
    LockManagerSnapshot: base_config_1.TYPE,
    LockOptions: base_config_1.TYPE,
    MIDIConnectionEventInit: base_config_1.TYPE,
    MIDIMessageEventInit: base_config_1.TYPE,
    MIDIOptions: base_config_1.TYPE,
    MediaCapabilitiesDecodingInfo: base_config_1.TYPE,
    MediaCapabilitiesEncodingInfo: base_config_1.TYPE,
    MediaCapabilitiesInfo: base_config_1.TYPE,
    MediaConfiguration: base_config_1.TYPE,
    MediaDecodingConfiguration: base_config_1.TYPE,
    MediaElementAudioSourceOptions: base_config_1.TYPE,
    MediaEncodingConfiguration: base_config_1.TYPE,
    MediaEncryptedEventInit: base_config_1.TYPE,
    MediaImage: base_config_1.TYPE,
    MediaKeyMessageEventInit: base_config_1.TYPE,
    MediaKeySystemConfiguration: base_config_1.TYPE,
    MediaKeySystemMediaCapability: base_config_1.TYPE,
    MediaMetadataInit: base_config_1.TYPE,
    MediaPositionState: base_config_1.TYPE,
    MediaQueryListEventInit: base_config_1.TYPE,
    MediaRecorderOptions: base_config_1.TYPE,
    MediaSessionActionDetails: base_config_1.TYPE,
    MediaStreamAudioSourceOptions: base_config_1.TYPE,
    MediaStreamConstraints: base_config_1.TYPE,
    MediaStreamTrackEventInit: base_config_1.TYPE,
    MediaTrackCapabilities: base_config_1.TYPE,
    MediaTrackConstraintSet: base_config_1.TYPE,
    MediaTrackConstraints: base_config_1.TYPE,
    MediaTrackSettings: base_config_1.TYPE,
    MediaTrackSupportedConstraints: base_config_1.TYPE,
    MessageEventInit: base_config_1.TYPE,
    MouseEventInit: base_config_1.TYPE,
    MultiCacheQueryOptions: base_config_1.TYPE,
    MutationObserverInit: base_config_1.TYPE,
    NavigationPreloadState: base_config_1.TYPE,
    NotificationAction: base_config_1.TYPE,
    NotificationOptions: base_config_1.TYPE,
    OfflineAudioCompletionEventInit: base_config_1.TYPE,
    OfflineAudioContextOptions: base_config_1.TYPE,
    OptionalEffectTiming: base_config_1.TYPE,
    OscillatorOptions: base_config_1.TYPE,
    PageTransitionEventInit: base_config_1.TYPE,
    PannerOptions: base_config_1.TYPE,
    PaymentCurrencyAmount: base_config_1.TYPE,
    PaymentDetailsBase: base_config_1.TYPE,
    PaymentDetailsInit: base_config_1.TYPE,
    PaymentDetailsModifier: base_config_1.TYPE,
    PaymentDetailsUpdate: base_config_1.TYPE,
    PaymentItem: base_config_1.TYPE,
    PaymentMethodChangeEventInit: base_config_1.TYPE,
    PaymentMethodData: base_config_1.TYPE,
    PaymentRequestUpdateEventInit: base_config_1.TYPE,
    PaymentValidationErrors: base_config_1.TYPE,
    Pbkdf2Params: base_config_1.TYPE,
    PerformanceMarkOptions: base_config_1.TYPE,
    PerformanceMeasureOptions: base_config_1.TYPE,
    PerformanceObserverInit: base_config_1.TYPE,
    PeriodicWaveConstraints: base_config_1.TYPE,
    PeriodicWaveOptions: base_config_1.TYPE,
    PermissionDescriptor: base_config_1.TYPE,
    PictureInPictureEventInit: base_config_1.TYPE,
    PointerEventInit: base_config_1.TYPE,
    PopStateEventInit: base_config_1.TYPE,
    PositionOptions: base_config_1.TYPE,
    ProgressEventInit: base_config_1.TYPE,
    PromiseRejectionEventInit: base_config_1.TYPE,
    PropertyIndexedKeyframes: base_config_1.TYPE,
    PublicKeyCredentialCreationOptions: base_config_1.TYPE,
    PublicKeyCredentialDescriptor: base_config_1.TYPE,
    PublicKeyCredentialEntity: base_config_1.TYPE,
    PublicKeyCredentialParameters: base_config_1.TYPE,
    PublicKeyCredentialRequestOptions: base_config_1.TYPE,
    PublicKeyCredentialRpEntity: base_config_1.TYPE,
    PublicKeyCredentialUserEntity: base_config_1.TYPE,
    PushSubscriptionJSON: base_config_1.TYPE,
    PushSubscriptionOptionsInit: base_config_1.TYPE,
    QueuingStrategy: base_config_1.TYPE,
    QueuingStrategyInit: base_config_1.TYPE,
    RTCAnswerOptions: base_config_1.TYPE,
    RTCCertificateExpiration: base_config_1.TYPE,
    RTCConfiguration: base_config_1.TYPE,
    RTCDTMFToneChangeEventInit: base_config_1.TYPE,
    RTCDataChannelEventInit: base_config_1.TYPE,
    RTCDataChannelInit: base_config_1.TYPE,
    RTCDtlsFingerprint: base_config_1.TYPE,
    RTCEncodedAudioFrameMetadata: base_config_1.TYPE,
    RTCEncodedVideoFrameMetadata: base_config_1.TYPE,
    RTCErrorEventInit: base_config_1.TYPE,
    RTCErrorInit: base_config_1.TYPE,
    RTCIceCandidateInit: base_config_1.TYPE,
    RTCIceCandidatePairStats: base_config_1.TYPE,
    RTCIceServer: base_config_1.TYPE,
    RTCInboundRtpStreamStats: base_config_1.TYPE,
    RTCLocalSessionDescriptionInit: base_config_1.TYPE,
    RTCOfferAnswerOptions: base_config_1.TYPE,
    RTCOfferOptions: base_config_1.TYPE,
    RTCOutboundRtpStreamStats: base_config_1.TYPE,
    RTCPeerConnectionIceErrorEventInit: base_config_1.TYPE,
    RTCPeerConnectionIceEventInit: base_config_1.TYPE,
    RTCReceivedRtpStreamStats: base_config_1.TYPE,
    RTCRtcpParameters: base_config_1.TYPE,
    RTCRtpCapabilities: base_config_1.TYPE,
    RTCRtpCodecCapability: base_config_1.TYPE,
    RTCRtpCodecParameters: base_config_1.TYPE,
    RTCRtpCodingParameters: base_config_1.TYPE,
    RTCRtpContributingSource: base_config_1.TYPE,
    RTCRtpEncodingParameters: base_config_1.TYPE,
    RTCRtpHeaderExtensionCapability: base_config_1.TYPE,
    RTCRtpHeaderExtensionParameters: base_config_1.TYPE,
    RTCRtpParameters: base_config_1.TYPE,
    RTCRtpReceiveParameters: base_config_1.TYPE,
    RTCRtpSendParameters: base_config_1.TYPE,
    RTCRtpStreamStats: base_config_1.TYPE,
    RTCRtpSynchronizationSource: base_config_1.TYPE,
    RTCRtpTransceiverInit: base_config_1.TYPE,
    RTCSentRtpStreamStats: base_config_1.TYPE,
    RTCSessionDescriptionInit: base_config_1.TYPE,
    RTCStats: base_config_1.TYPE,
    RTCTrackEventInit: base_config_1.TYPE,
    RTCTransportStats: base_config_1.TYPE,
    ReadableStreamGetReaderOptions: base_config_1.TYPE,
    ReadableStreamReadDoneResult: base_config_1.TYPE,
    ReadableStreamReadValueResult: base_config_1.TYPE,
    ReadableWritablePair: base_config_1.TYPE,
    RegistrationOptions: base_config_1.TYPE,
    RequestInit: base_config_1.TYPE,
    ResizeObserverOptions: base_config_1.TYPE,
    ResponseInit: base_config_1.TYPE,
    RsaHashedImportParams: base_config_1.TYPE,
    RsaHashedKeyAlgorithm: base_config_1.TYPE,
    RsaHashedKeyGenParams: base_config_1.TYPE,
    RsaKeyAlgorithm: base_config_1.TYPE,
    RsaKeyGenParams: base_config_1.TYPE,
    RsaOaepParams: base_config_1.TYPE,
    RsaOtherPrimesInfo: base_config_1.TYPE,
    RsaPssParams: base_config_1.TYPE,
    SVGBoundingBoxOptions: base_config_1.TYPE,
    ScrollIntoViewOptions: base_config_1.TYPE,
    ScrollOptions: base_config_1.TYPE,
    ScrollToOptions: base_config_1.TYPE,
    SecurityPolicyViolationEventInit: base_config_1.TYPE,
    ShadowRootInit: base_config_1.TYPE,
    ShareData: base_config_1.TYPE,
    SpeechSynthesisErrorEventInit: base_config_1.TYPE,
    SpeechSynthesisEventInit: base_config_1.TYPE,
    StaticRangeInit: base_config_1.TYPE,
    StereoPannerOptions: base_config_1.TYPE,
    StorageEstimate: base_config_1.TYPE,
    StorageEventInit: base_config_1.TYPE,
    StreamPipeOptions: base_config_1.TYPE,
    StructuredSerializeOptions: base_config_1.TYPE,
    SubmitEventInit: base_config_1.TYPE,
    TextDecodeOptions: base_config_1.TYPE,
    TextDecoderOptions: base_config_1.TYPE,
    TextEncoderEncodeIntoResult: base_config_1.TYPE,
    TouchEventInit: base_config_1.TYPE,
    TouchInit: base_config_1.TYPE,
    TrackEventInit: base_config_1.TYPE,
    Transformer: base_config_1.TYPE,
    TransitionEventInit: base_config_1.TYPE,
    UIEventInit: base_config_1.TYPE,
    ULongRange: base_config_1.TYPE,
    UnderlyingByteSource: base_config_1.TYPE,
    UnderlyingDefaultSource: base_config_1.TYPE,
    UnderlyingSink: base_config_1.TYPE,
    UnderlyingSource: base_config_1.TYPE,
    ValidityStateFlags: base_config_1.TYPE,
    VideoColorSpaceInit: base_config_1.TYPE,
    VideoConfiguration: base_config_1.TYPE,
    VideoFrameCallbackMetadata: base_config_1.TYPE,
    WaveShaperOptions: base_config_1.TYPE,
    WebGLContextAttributes: base_config_1.TYPE,
    WebGLContextEventInit: base_config_1.TYPE,
    WheelEventInit: base_config_1.TYPE,
    WindowPostMessageOptions: base_config_1.TYPE,
    WorkerOptions: base_config_1.TYPE,
    WorkletOptions: base_config_1.TYPE,
    NodeFilter: base_config_1.TYPE_VALUE,
    XPathNSResolver: base_config_1.TYPE,
    ANGLE_instanced_arrays: base_config_1.TYPE,
    ARIAMixin: base_config_1.TYPE,
    AbortController: base_config_1.TYPE_VALUE,
    AbortSignalEventMap: base_config_1.TYPE,
    AbortSignal: base_config_1.TYPE_VALUE,
    AbstractRange: base_config_1.TYPE_VALUE,
    AbstractWorkerEventMap: base_config_1.TYPE,
    AbstractWorker: base_config_1.TYPE,
    AnalyserNode: base_config_1.TYPE_VALUE,
    Animatable: base_config_1.TYPE,
    AnimationEventMap: base_config_1.TYPE,
    Animation: base_config_1.TYPE_VALUE,
    AnimationEffect: base_config_1.TYPE_VALUE,
    AnimationEvent: base_config_1.TYPE_VALUE,
    AnimationFrameProvider: base_config_1.TYPE,
    AnimationPlaybackEvent: base_config_1.TYPE_VALUE,
    AnimationTimeline: base_config_1.TYPE_VALUE,
    Attr: base_config_1.TYPE_VALUE,
    AudioBuffer: base_config_1.TYPE_VALUE,
    AudioBufferSourceNode: base_config_1.TYPE_VALUE,
    AudioContext: base_config_1.TYPE_VALUE,
    AudioDestinationNode: base_config_1.TYPE_VALUE,
    AudioListener: base_config_1.TYPE_VALUE,
    AudioNode: base_config_1.TYPE_VALUE,
    AudioParam: base_config_1.TYPE_VALUE,
    AudioParamMap: base_config_1.TYPE_VALUE,
    AudioProcessingEvent: base_config_1.TYPE_VALUE,
    AudioScheduledSourceNodeEventMap: base_config_1.TYPE,
    AudioScheduledSourceNode: base_config_1.TYPE_VALUE,
    AudioWorklet: base_config_1.TYPE_VALUE,
    AudioWorkletNodeEventMap: base_config_1.TYPE,
    AudioWorkletNode: base_config_1.TYPE_VALUE,
    AuthenticatorAssertionResponse: base_config_1.TYPE_VALUE,
    AuthenticatorAttestationResponse: base_config_1.TYPE_VALUE,
    AuthenticatorResponse: base_config_1.TYPE_VALUE,
    BarProp: base_config_1.TYPE_VALUE,
    BaseAudioContextEventMap: base_config_1.TYPE,
    BaseAudioContext: base_config_1.TYPE_VALUE,
    BeforeUnloadEvent: base_config_1.TYPE_VALUE,
    BiquadFilterNode: base_config_1.TYPE_VALUE,
    Blob: base_config_1.TYPE_VALUE,
    BlobEvent: base_config_1.TYPE_VALUE,
    Body: base_config_1.TYPE,
    BroadcastChannelEventMap: base_config_1.TYPE,
    BroadcastChannel: base_config_1.TYPE_VALUE,
    ByteLengthQueuingStrategy: base_config_1.TYPE_VALUE,
    CDATASection: base_config_1.TYPE_VALUE,
    CSSAnimation: base_config_1.TYPE_VALUE,
    CSSConditionRule: base_config_1.TYPE_VALUE,
    CSSContainerRule: base_config_1.TYPE_VALUE,
    CSSCounterStyleRule: base_config_1.TYPE_VALUE,
    CSSFontFaceRule: base_config_1.TYPE_VALUE,
    CSSFontFeatureValuesRule: base_config_1.TYPE_VALUE,
    CSSFontPaletteValuesRule: base_config_1.TYPE_VALUE,
    CSSGroupingRule: base_config_1.TYPE_VALUE,
    CSSImportRule: base_config_1.TYPE_VALUE,
    CSSKeyframeRule: base_config_1.TYPE_VALUE,
    CSSKeyframesRule: base_config_1.TYPE_VALUE,
    CSSLayerBlockRule: base_config_1.TYPE_VALUE,
    CSSLayerStatementRule: base_config_1.TYPE_VALUE,
    CSSMediaRule: base_config_1.TYPE_VALUE,
    CSSNamespaceRule: base_config_1.TYPE_VALUE,
    CSSPageRule: base_config_1.TYPE_VALUE,
    CSSRule: base_config_1.TYPE_VALUE,
    CSSRuleList: base_config_1.TYPE_VALUE,
    CSSStyleDeclaration: base_config_1.TYPE_VALUE,
    CSSStyleRule: base_config_1.TYPE_VALUE,
    CSSStyleSheet: base_config_1.TYPE_VALUE,
    CSSSupportsRule: base_config_1.TYPE_VALUE,
    CSSTransition: base_config_1.TYPE_VALUE,
    Cache: base_config_1.TYPE_VALUE,
    CacheStorage: base_config_1.TYPE_VALUE,
    CanvasCaptureMediaStreamTrack: base_config_1.TYPE_VALUE,
    CanvasCompositing: base_config_1.TYPE,
    CanvasDrawImage: base_config_1.TYPE,
    CanvasDrawPath: base_config_1.TYPE,
    CanvasFillStrokeStyles: base_config_1.TYPE,
    CanvasFilters: base_config_1.TYPE,
    CanvasGradient: base_config_1.TYPE_VALUE,
    CanvasImageData: base_config_1.TYPE,
    CanvasImageSmoothing: base_config_1.TYPE,
    CanvasPath: base_config_1.TYPE,
    CanvasPathDrawingStyles: base_config_1.TYPE,
    CanvasPattern: base_config_1.TYPE_VALUE,
    CanvasRect: base_config_1.TYPE,
    CanvasRenderingContext2D: base_config_1.TYPE_VALUE,
    CanvasShadowStyles: base_config_1.TYPE,
    CanvasState: base_config_1.TYPE,
    CanvasText: base_config_1.TYPE,
    CanvasTextDrawingStyles: base_config_1.TYPE,
    CanvasTransform: base_config_1.TYPE,
    CanvasUserInterface: base_config_1.TYPE,
    ChannelMergerNode: base_config_1.TYPE_VALUE,
    ChannelSplitterNode: base_config_1.TYPE_VALUE,
    CharacterData: base_config_1.TYPE_VALUE,
    ChildNode: base_config_1.TYPE,
    ClientRect: base_config_1.TYPE,
    Clipboard: base_config_1.TYPE_VALUE,
    ClipboardEvent: base_config_1.TYPE_VALUE,
    ClipboardItem: base_config_1.TYPE_VALUE,
    CloseEvent: base_config_1.TYPE_VALUE,
    Comment: base_config_1.TYPE_VALUE,
    CompositionEvent: base_config_1.TYPE_VALUE,
    ConstantSourceNode: base_config_1.TYPE_VALUE,
    ConvolverNode: base_config_1.TYPE_VALUE,
    CountQueuingStrategy: base_config_1.TYPE_VALUE,
    Credential: base_config_1.TYPE_VALUE,
    CredentialsContainer: base_config_1.TYPE_VALUE,
    Crypto: base_config_1.TYPE_VALUE,
    CryptoKey: base_config_1.TYPE_VALUE,
    CustomElementRegistry: base_config_1.TYPE_VALUE,
    CustomEvent: base_config_1.TYPE_VALUE,
    DOMException: base_config_1.TYPE_VALUE,
    DOMImplementation: base_config_1.TYPE_VALUE,
    DOMMatrix: base_config_1.TYPE_VALUE,
    SVGMatrix: base_config_1.TYPE_VALUE,
    WebKitCSSMatrix: base_config_1.TYPE_VALUE,
    DOMM
